tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -17,7 +17,7 @@
     "H_{XY} = \\frac{1}{2}\\sum_{i<j} \\frac{C_3}{R_{ij}^3}(\\sigma^+_i \\sigma^-_j + \\sigma^-_i \\sigma^+_j) \n",
     "$$\n",
     "\n",
-    "where $\\sigma^{±} = (\\sigma^x ± i\\sigma^y)/2$, $R_{ij}$ is the distance between atom $i$ and atom $j$, and $C_3$ is a coefficient dependent on the device and taken to 3070 MHz $\\cdot \\mu m^{-3}$ for the `MockDevice`.\n",
+    "where $\\sigma^{±} = (\\sigma^x ± i\\sigma^y)/2$, $R_{ij}$ is the distance between atom $i$ and atom $j$, and $C_3$ is a coefficient dependent on the device and taken as  $ \\frac{C_3}{\\hbar} = 3070$ MHz $\\cdot \\mu m^3$ for the `MockDevice`.\n",
```

Comment:
Good catch!

Reply:
Actually the value seems to be also different from the one defined in _mock_device.py
(3070 in the notebook, 3700 in MockDevice). I have just commited this

Reply:
Yes, and perhaps for the units I would use `/µs` rather than `MHz` to highlight there is a factor of `2π` there (ie the difference between `C3/h` vs `C3/hbar`)

Reply:
Yes I agree, it looks a bit more barbarian but this way the notebook and the API Reference have the same expression. I am doing it.